tools/harness-automation/autothreadharness/settings_sample.py

```diff
@@ -59,16 +59,19 @@
 THREAD_SED_POLLING_INTERVAL = 0
 """int: SED polling interval in seconds. Set to 0 to use Harness's default value."""
 
-HARNESS_HOME = 'C:\\GRL\\Thread1.1'
+HARNESS_HOME = "C:\\GRL\\Thread1.1"
 r"""str: Harness installation path, e.g. ``C:\GRL\Thread1.1``"""
 
-HARNESS_URL = 'http://127.0.0.1:8000'
+HARNESS_URL = "http://127.0.0.1:8000"
 """str: Harness front-end url"""
 
-TESTER_NAME = 'Thread Open'
+HARNESS_VERSION = "R1.4.0"
+"""str: Harness version, official or internal release, e.g. 'R1.4.0' or 'V49.4' """
+
+TESTER_NAME = "Thread Open"
```

Comment:
Is `TESTER_NAME` still required? 
'AutoGeneratePdf = True' in Test Harness' configuration.ini should be able to auto generate the pdf report now, right?

Reply:
TESTER_NAME is used to set the TESTER value when save-pdf if version <= R1.4.0